### DISCUSSION TITLE
Lesson 4 : Error installing web3.py

### DISCUSSION THREAD

**Discussion by shucks18** [smartcontractkit/full-blockchain-solidity-course-py#906] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/81901871/151743929-dbf13d07-99be-4c03-8667-dd1a702b0236.png)
![image](https://user-images.githubusercontent.com/81901871/151743953-201b76b2-2f59-422b-932c-406bcf7a7901.png)
![image](https://user-images.githubusercontent.com/81901871/151743976-f4073d78-d01a-4eeb-8300-44f5f0ea0f89.png)

**Comment by shucks18** [comment, marked as answer]
https://www.youtube.com/watch?v=rcI1_e38BWs&ab_channel=AdityaThakur

**Reply by cromewar** [reply]
Nice catch, I should add that video here: https://medium.com/@cromewar/the-mighty-cytools-error-at-web3-smart-contract-development-with-brownie-63335d50f230
